frontends/election-manager/src/app.test.tsx

```diff
@@ -339,7 +340,8 @@ test('L&A (logic and accuracy) flow', async () => {
   expect(mockKiosk.log).toHaveBeenCalledWith(
     expect.stringContaining(LogEventId.TallyReportPrinted)
   );
-  expect(screen.getAllByText('0').length).toBe(40);
+  const zeroReport = document.querySelector('#zero-report') as HTMLElement;
+  expect(domGetAllByText(zeroReport, '0').length).toBe(40);
```

Comment:
Now that another report is rendered on the same screen, we have to isolate the zero report to test its contents.